### PR TITLE
Add forwardRef to Box and Flex

### PR DESCRIPTION
**Pull request by Malinicz**
Without `forwardRef` we are not able to use `ref` on those elements

### CODE REVIEW

**Review by ElaLukaszewicz — APPROVED** [review]

File: src/components/accordion/Accordion.spec.jsx [hunk 1]
```diff
@@ -124,7 +125,7 @@ describe('<Accordion>', () => {
 
     expect(
       accordion
-        .find('Box')
+        .find(Box)
```
Comment (coderitual) [inline src/components/accordion/Accordion.spec.jsx:128]: how about setting Box.displayName = "Box", if I understand correctly it uses displayName to find component without a need of importing real component

Comment (Malinicz) [inline src/components/accordion/Accordion.spec.jsx:128]: this is something worth to remember, when forwardRef is used on element we can't find by string in tests :)

Comment (Malinicz) [reply src/components/accordion/Accordion.spec.jsx:128]: good point 🤔  Haven't thought about it, but actually this seems like a better solution

Comment (coderitual) [reply src/components/accordion/Accordion.spec.jsx]: @Malinicz you can wrap setting display name in env check and only set in on development /tests.

something like: 

```
if(dev) {
...
}

where dev is ofc check for `process.env.NODE_ENV`
```

Comment (Malinicz) [reply src/components/accordion/Accordion.spec.jsx]: yeah but then I would have the same problem in brainly-frontend, right?

**Review by bartoszlorek — APPROVED** [review]

File: src/components/accordion/Accordion.spec.jsx (hunk 1 above)
Comment (Malinicz) [reply src/components/accordion/Accordion.spec.jsx]: and if I just set displayName on component, I also would be able to find by string in our repo, or I'm wrong @coderitual?

Comment (coderitual) [reply src/components/accordion/Accordion.spec.jsx]: @Malinicz you should be able to find it in our repo too as long as we don't set production flag for tests in brainly frontend. This code will be present in production build of style guide and only removed on production build of application uses style guide.

Comment (coderitual) [reply src/components/accordion/Accordion.spec.jsx]: @Malinicz So setting `export const __DEV__ = process.env.NODE_ENV !== "production"` in some utils file and importing it into component file should work. This also should work with application which uses our component. So in brainly frontend we can test component using strings as well

**Review by matzimowski — APPROVED** [review]

**Review by coderitual — APPROVED** [review]

**Pull request merged**